File: src/001. Multiples of 3 and 5/001.java
import java.util.stream.IntStream;
public class Main {

    public static void main(String[] args) {
        int sum = IntStream.range(1, 1000)
                .filter(n -> n % 3 == 0 || n % 5 == 0)
                .sum();
        System.out.println(sum);
    }
}

File: src/001. Multiples of 3 and 5/001.cpp
#include <iostream>

int main(void) {
    int sum = 0;
    for (int i = 0; i < 1000; i++) {
        if (i % 3 == 0 || i % 5 == 0) {
            sum += i;
        }
    }
    std::cout << sum;
}
